packages/ods-react/src/components/combobox/src/controller/combobox.ts

```diff
@@ -168,5 +159,6 @@ export {
   flattenItems,
   findLabelForValue,
   getFilteredItems,
+  splitTextBySearchTerm,
   type ComboboxProps,
```

Comment:
is it actually used outside of controller? May need to rename as it is confusing with `ComboboxProp` from context